### PR TITLE
add local planb-tokeninfo

### PR DESCRIPTION
**Pull request by szuecs**
Add local planb-tokeninfo to remove the rtt to an external tokeninfo service.

### CODE REVIEW

**Comment by tuxlife** [comment]
:+1:

File: runtime/etc/init/planb-tokeninfo.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description "PlanB Tokeninfo"
+
+stop on shutdown
+
+respawn
+
+script
+	build/planb-tokeninfo
```
Comment (hjacobs) [inline runtime/etc/init/planb-tokeninfo.conf]: is "build" the right path? I guess not?

**Comment by lukasniemeier-zalando** [comment]
Cool, is this already released?

**Comment by szuecs** [comment]
@lukasniemeier-zalando no, but we plan to release it today

**Comment by hjacobs** [comment]
This does not work as expected yet --- we should also set the `TOKENINFO_URL` env var to "http://localhost:9021/oauth2/tokeninfo" for the application Docker container.

**Comment by szuecs** [comment]
@hjacobs why? If I understand it correctly the user can and should define it herself.

**Comment by hjacobs** [comment]
@szuecs we set `TOKENINFO_URL` in Taupage, so we should automatically set it to the correct localhost URL when starting it locally. The user should not be aware on which port it runs locally...

**Comment by szuecs** [comment]
@hjacobs I will set it in our Zalando specific configuration, otherwise it would not make sense.

**Pull request merged**